src/presets/fancy.ts

```diff
@@ -12,6 +12,14 @@ const getDateTimeString = (date: Date) => {
   return `${day}/${month}/${year} ${hours}:${minutes}:${seconds}`;
 };
 
+const formatDuration = (duration: number) => {
+  if (duration < 1000) return `${duration}μs`;
+  else if (duration < 1000000) return `${(duration / 1000).toFixed(2)}ms`;
+  else if (duration < 60000000) return `${(duration / 1000000).toFixed(2)}s`;
+  else if (duration < 3600000000) return `${(duration / 60000000).toFixed(2)}m`;
```

Comment:
For readability, I suggest doing `60 * 60 * 1_000_000` which makes it clear how `360000000000` came from.